### ISSUE TITLE
Kotlin support

### ISSUE THREAD

**Issue by Vampire** [lppedd/idea-debugger-enhancer#1]
Please also add Kotlin support.
I see the properties in Java and Groovy line breakpoints, but not in Kotlin line breakpoints yet.

**Comment by lppedd** [comment]
@Vampire yeah, Kotlin has different breakpoints implementation unfortunately.
I'll see what I can do about it! Sorry for the inconvenience

**Comment by Vampire** [comment]
Don't be stupid, you're my hero. :-)
Kotlin just happens to be one of my main languages currently if I do Gradle related things so it happened to be a Kotlin file where I tried it first. :-D

**Comment by lppedd** [comment]
@Vampire I've done it. Need to polish, then I'll commit and publish.

**Comment by Vampire** [comment]
How is the polishing going? :-)

**Comment by lppedd** [comment]
@Vampire hey! I didn't manage to work on this during the week. I'll try to get it done and published soon

**Comment by Vampire** [comment]
Any chance on this really getting out at some point?
I needed it two times in the meantime. :-(

**Comment by Vampire** [comment]
Maybe?

**Comment by okmyan** [comment]
Hello! I found your plugin yesterday, and it's something incredible! This is a very useful thing for me. Thank you very much for this plugin! However, in my working project, 80% of the code is written in kotlin. I would really like to see an extended version of the plugin with kotlin support... I hope you will post this version. My team would really use such a plugin!
